node/src/components/block_accumulator/block_acceptor.rs

```diff
@@ -93,7 +93,6 @@ impl BlockAcceptor {
         }
 
         if self.block.is_none() {
-            self.touch();
```

Comment:
So this `touch()` could be removed because we now touch only [here](https://github.com/casper-network/casper-node/blob/91e05f4ab21bfee79bcd9ce9de6cfa8eba055983/node/src/components/block_accumulator/block_acceptor.rs#L207), right?

Reply:
Yes, and here we don't know yet whether the block is sufficiently signed.